other-guides/hosting-on-linux-server

```diff
@@ -0,0 +1,53 @@
+# Hosting on a Linux Server
+
+The best way to host your bot is on a Linux Server (or simply said: just a virtual server. Its also for bare metal servers).
+
+## Requirements
+
+- A Linux Server with a stable version, node, npm, git, pm2. 
+- Github repository from your bot (privat).
```

Comment:
GitHub isn't needed, people can use other services (Docker, GitLab, BitBucket, SourceForge, Gitea, self-hosted...), or even, upload the files directly.